docs/Toy/toy/parser.py

```diff
@@ -119,7 +119,7 @@ def parseModule(self):
         # If we didn't reach EOF, there was an error during parsing
         self.pop_token(EOFToken)
 
-        return ModuleAST(functions)
+        return ModuleAST(tuple(functions))
```

Comment:
For some reason, the type checker was happy with this but not the test code passing a manually constructed list, due to the `list` generic parameter not being covariant. Tuples are immutable, which makes the type checker happy